Tests/SentryTests/Integrations/Performance/IO/SentryNSDataTrackerTests.swift

```diff
@@ -0,0 +1,273 @@
+        let data = try? sut.measureNSData(fromFile: self.fixture.filePath, options: .uncached) { path, options, _ -> Data in
+            span = self.firstSpan(transaction)
+            usedOptions = options
+            usedPath = path
+            return self.fixture.data
+        }
+        
+        XCTAssertEqual(usedPath, fixture.filePath)
+        XCTAssertEqual(data?.count, fixture.data.count)
+        XCTAssertEqual(usedOptions, .uncached)
+        
+        assertDataSpan(span, path: fixture.filePath, operation: SENTRY_FILE_READ_OPERATION, size: fixture.data.count)
+    }
+    
+    func testReadFromURLOptionsError() {
+        let sut = fixture.getSut()
+        let transaction = SentrySDK.startTransaction(name: "Transaction", operation: "Test", bindToScope: true)
+        var span: Span?
+        var usedUrl: URL?
+        let url = URL(fileURLWithPath: fixture.filePath)
+        var usedOptions: NSData.ReadingOptions?
+        
+        let data = try? sut.measureNSData(from: url, options: .uncached) { url, options, _ in
+            span = self.firstSpan(transaction)
+            usedOptions = options
+            usedUrl = url
+            return self.fixture.data
+        }
+        
+        XCTAssertEqual(usedUrl, url)
+        XCTAssertEqual(data?.count, fixture.data.count)
+        XCTAssertEqual(usedOptions, .uncached)
+        
+        assertDataSpan(span, path: url.path, operation: SENTRY_FILE_READ_OPERATION, size: fixture.data.count)
+    }
+    
+    func testDontTrackSentryFilesRead() {
+        let sut = fixture.getSut()
+        let transaction = SentrySDK.startTransaction(name: "Transaction", operation: "Test", bindToScope: true)
+        var span: Span?
+       
+        let expect = expectation(description: "")
+        let _ = sut.measureNSData(fromFile: fixture.sentryPath) { _ in
+            span = self.firstSpan(transaction)
+            expect.fulfill()
+            return nil
+        }
+        
+        XCTAssertNil(span)
+        wait(for: [expect], timeout: 0.1)
+    }
+    
+    func testTransactionDescription() {
+        let sut = Dynamic(fixture.getSut())
+        var result = sut.transactionDescription(forFile: "file/test", fileSize: 12) as String?
+        XCTAssertEqual(result, "test (12 bytes)")
+        
+        result = sut.transactionDescription(forFile: "file/test", fileSize: 0) as String?
+        XCTAssertEqual(result, "test")
+        
+        result = sut.transactionDescription(forFile: "file\test", fileSize: 12) as String?
+        XCTAssertEqual(result, "file\test (12 bytes)")
+        
+        //iOS converter does not use the more common file size measurament of 1024 bytes equal 1KB
+        result = sut.transactionDescription(forFile: "file/test", fileSize: 1_024) as String?
+        XCTAssertEqual(result, "test (1 KB)")
+        
+        result = sut.transactionDescription(forFile: "file/test", fileSize: 1_024 * 1_023) as String?
+        XCTAssertEqual(result, "test (1,023 KB)")
```

Comment:
`m`: This test will fail on a computer with a German locale. We don't need to test `NSByteCountFormatter` here, so I would remove the test.

Reply:
I created a custom formatter, now we are safe.

Reply:
As discussed, I think for the alpha we don't need a custom formatter. We can just remove the test.